certified-connectors/Nodefusion Portal/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "6f2a7cda-d35a-40ac-b913-d3ec4066c42a",
+          "scopes": [
+            "https://3procloud.onmicrosoft.com/3proportal/user_impersonation"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://login.microsoftonline.com/te/3procloud.onmicrosoft.com/b2c_1a_signuporsigninwithaad/oauth2/v2.0/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://login.microsoftonline.com/te/3procloud.onmicrosoft.com/b2c_1a_signuporsigninwithaad/oauth2/v2.0/token"
+            },
+            "refreshUrl": {
+              "value": "https://login.microsoftonline.com/te/3procloud.onmicrosoft.com/b2c_1a_signuporsigninwithaad/oauth2/v2.0/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",

```

Comment:
@hkusulja You have accepted that your connector would not contain this iconBrandColor, can you please provide a new iconBrandColor ?

![image](https://user-images.githubusercontent.com/56276394/143951703-c20532ff-b81c-44e2-90fc-f10f13eda8d8.png)

Reply:
we fixed this